src/data/TeamData.mjs

```diff
@@ -420,7 +358,7 @@ export default {
 	},
 	zOnlyKroks: {
 		name: "zOnlyKroks",
-		discord: "zOnlyKroks | Finn",
+		discord: "zonlykroks",
 		github: "zOnlyKroks",
 		avatar: "/assets/img/team/zOnlyKroks.png",
```

Comment:
Avatar was updated to
![](https://cdn.discordapp.com/avatars/394770112139493376/9d6d239a2070737a3eb189c80a167f0a.webp?size=128)